examples/auth/envoy_config.json

```diff
@@ -443,6 +443,62 @@
                               }
                             }
                           },
+                          {
+                            "decorator": {
+                              "operation": "ingress UnknownOperationName"
+                            },
+                            "directResponse": {
+                              "body": {
+                                "inlineString": "The current request is matched to defined url template \"/shelves\" but the http method is not allowed"
```

Comment:
I feel Not need to print url template in the error message.  Just
The request path is defined in the API, but http method is not supported

Reply:
IMHO, it is useful for users to debug by telling them which uri template the request is mapped to.

Reply:
SG